pkg/gofr/external_db.go

```diff
@@ -156,6 +157,23 @@ func (a *App) AddScyllaDB(db container.ScyllaDBProvider) {
 	a.container.ScyllaDB = db
 }
 
+// AddArango sets the ArangoDB datasource in the app's container.
+func (a *App) AddArango(db container.ArangoProvider) {
```

Comment:
```suggestion
// AddArangoDB sets the ArangoDB datasource in the app's container.
func (a *App) AddArangoDB(db container.ArangoProvider) {
```
This was left behind